### PR TITLE
Bump Docker Build to Node 12.18.3

### PR DESCRIPTION
**Pull request by alexm118**
Update our node version to 12.18.3 in our docker build job, also updating the run command to open a pseudo TTY.

## Checklist

- [ ] PR has an informative and human-readable title
  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
- [ ] Changes are limited to a single goal (no scope creep)
- [ ] Code follows the standards laid out in the [CFPB development guidelines](https://github.com/cfpb/development)
- [ ] Future todos are captured in comments and/or tickets
- [ ] Project documentation has been updated, potentially one or more of:
  - [This repo’s docs](https://cfpb.github.io/cfgov-refresh/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance

#### Browser testing

Visually tested in the following supported browsers:
- [ ] Firefox
- [ ] Chrome
- [ ] Safari
- [ ] Edge 18 (the last Edge prior to it switching to Chromium)
- [ ] Internet Explorer 11 and 8 (via emulation in 11's dev tools)
- [ ] Safari on iOS
- [ ] Chrome on Android

#### Accessibility

- [ ] Keyboard friendly (navigable with tab, space, enter, arrow keys, etc.)
- [ ] Screen reader friendly
- [ ] Does not introduce new errors or warnings in [WAVE](https://wave.webaim.org/extension/)

#### Other

- [ ] Is useable without CSS
- [ ] Is useable without JS
- [ ] Does not introduce new lint warnings
- [ ] Flexible from small to large screens

### CODE REVIEW

File: docker/deployable-zipfile/build.sh [hunk 1]
```diff
@@ -6,6 +6,7 @@ docker build -t cfgov-deployable-zipfile-builder docker/deployable-zipfile
 
 docker run \
   --rm \
+  -it \
```
[the remark this answers is not in the document]
Comment (alexm118) [reply docker/deployable-zipfile/build.sh:9]: From everything I've read there really isn't a downside and docker typically recommends operating this way

**Comment by willbarton** [comment]
@alexm118 This looks good. We can try building it before merging, by running the job with the `origin/add-exit-code` ref.

**Comment by willbarton** [comment]
> A suggestion is to also change node from 10 to 12 here: https://github.com/cfpb/cfgov-refresh/blob/master/Dockerfile#L28

@cwdavies I think the point here is not to do the upgrade (there's another PR for that, #5789), but just to see if we can build the artifact with Node 12 using Docker without issue (since we can't install Node 12 on Jenkins with the version of RHEL they're running).

**Comment by alexm118** [comment]
@willbarton The test run [passed](https://zusa.demo.cfpb.gov/job/cf.gov-ansible-docker-build/711/) and ran in under 9 minutes - lets see how it works in action.

**Pull request merged**